src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -521,19 +528,11 @@ private boolean addProvisionedSlave(@Nonnull PodTemplate template, @CheckForNull
                     .filter(x -> x.getStatus().getPhase().toLowerCase().matches("(running|pending)"))
                     .collect(Collectors.toList());
         }
-
-        if (allActiveSlavePods != null && containerCap <= allActiveSlavePods.size() + scheduledCount) {
-            LOGGER.log(Level.INFO,
-                    "Total container cap of {0} reached, not provisioning: {1} running or pending in namespace {2} with Kubernetes labels {3}",
-                    new Object[] { containerCap, allActiveSlavePods.size() + scheduledCount, templateNamespace, getLabels() });
-            return false;
-        }
-
         if (activeTemplateSlavePods != null && allActiveSlavePods != null && template.getInstanceCap() <= activeTemplateSlavePods.size() + scheduledCount) {
             LOGGER.log(Level.INFO,
-                    "Template instance cap of {0} reached for template {1}, not provisioning: {2} running or pending in namespace {3} with label \"{4}\" and Kubernetes labels {5}",
+                    "Maximum number of concurrently running agent pods ({0}) reached for template '{1}' in Kubernetes Cloud '{6}', not provisioning: {2} running or pending in namespace {3} with label \"{4}\" and Kubernetes labels {5}",
```

Comment:
talking from memory the single quotes will prevent the expansion of `{1}` and `{6}`

Reply:
Done.